app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -82,11 +83,17 @@ class RoutePresenter(
         return DataStoreAction.UpdateCredentials(credentials)
     }
 
+    @TargetApi(Build.VERSION_CODES.O)
```

Comment:
I don't think all of our routes should be gated on SDK >= 26! we are probably going to route places on lower versions as well :P